pages/linux/mycli.md

```diff
@@ -0,0 +1,15 @@
+# mycli
+
+> A CLI for MySQL, MariaDB, and Percona with auto-completion and syntax highlighting.
+
+- Connect to a database with current login user:
+
+`mycli {{database_name}}`
+
+- Connect to a database with specified user:
```

Comment:
Perhaps ...`with the specified`... would read better?

Reply:
It's right.

Reply:
Erm it's right? Right how?